docs/release-notes/cloud/cloud-2022-06.md

```diff
@@ -0,0 +1,79 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud June 2022.
+included_jira_versions: ['2022.Q3.1']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.1.19
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.1.55
+---
+
+# Cloud June 2022
+
+These release notes are for the Codacy Cloud updates during June 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Codacy now supports using [expiring access tokens](https://docs.gitlab.com/ee/integration/oauth_provider.html#expiring-access-tokens){: target="_blank"} to integrate with GitLab. Users affected by issues connecting to GitLab should re-login on the Codacy UI **using their GitLab accounts**, or [revoke the GitLab integration on Codacy](../../getting-started/which-permissions-does-codacy-need-from-my-account.md#revoking-access-to-integrations) if the issues persist. (CY-6117)
+-   [Pull request notification emails](../../account/emails.md#managing-your-email-notifications) now display the diff coverage metric. (CY-5700)
+
+## Bug fixes
+
+-   Fixed an issue to allow saving coverage data for the first commit in a repository. (CY-6196)
+-   Fixed an issue that could allow an attacker with an account on both the Codacy Self-hosted instance and on the Git provider to do destructive actions on **Codacy personal organizations**, such as removing members from Codacy or deleting the personal organization on Codacy. The issue didn't allow the attacker to see or change any information on the Git provider organization or repositories. CVSS v3.1 score: 7.9 (High) (CY-6184, CY-6187, CY-6188)
```

Comment:
@prcr I guess here we don't want the SH ones right?

Reply:
Yes, the release notes are the same as in [Codacy Self-hosted 8.1.0](https://docs.codacy.com/release-notes/self-hosted/self-hosted-v8.1.0/), but with the scores updated to apply to the Codacy Cloud environment.

Reply:
I would then just drop the part related to the SH instance the `both the Codacy Self-hosted instance`

Reply:
Good catch, I was focusing on including the correct CVSS score for Codacy Cloud and didn't notice that bit mentioning Codacy Self-hosted! :ok_hand: